### ISSUE TITLE
Book is down: SSL error when opening Github pages

### ISSUE THREAD

**Issue by corneliusroemer** [pola-rs/polars-book#261]
When opening https://pola-rs.github.io/polars-book/ I get an SSL error:

<img width="1463" alt="image" src="https://user-images.githubusercontent.com/25161793/221400411-fd6cd87f-e589-408c-b677-3bf91d62cce1.png">

**Comment by corneliusroemer** [comment]
Here's the most recent web.archive.org version of the guide, almost a year old but better than nothing: https://web.archive.org/web/20220527020732/https://pola-rs.github.io/polars-book/user-guide/index.html

Maybe one could automate saving to archive.org every month or so, in case something like this happens again in the future.

**Comment by corneliusroemer** [comment]
Curl gives the following error:

```console
$ curl https://pola-rs.github.io/polars-book/ --verbose  
*   Trying 185.199.111.153:443...
* Connected to pola-rs.github.io (185.199.111.153) port 443 (#0)
* ALPN: offers h2
* ALPN: offers http/1.1
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (OUT), TLS handshake, Client hello (1):
* error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
* Closing connection 0
curl: (35) error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
```

**Comment by ritchie46** [comment]
I can access it on my laptop and on mobile phone.

**Comment by corneliusroemer** [comment]
I've tried various browsers, and my phone, rebooted etc. and it looks like it's related to my home router/ISP. When I use mobile data/hotspot it works. Strange. I don't encounter this issue on any other websites, not even other Github pages.

I'll close as not reproducible.

**Comment by corneliusroemer** [comment]
Similar to this issue: https://stackoverflow.com/questions/73950564/err-ssl-protocol-and-ssl-error-rx-record-too-long-with-a-specific-isp-only

I got the error both on my Android phone (Firefox, Brave) and Mac (Firefox, Chrome, Brave) when connecting through ISP (Monzoon, Switzerland). All worked fine when connecting through mobile internet (yallo)

**Comment by corneliusroemer** [comment]
Figured it out! It was the stupid "AiProtection" feature of the ASUS router. Polars book got flagged as "phishing", clearly false positive. Will switch that damn feature off. But in any case, I've reported the site as safe.

Other users with the same router who've got "AiProtection" on will get the same error. So worth reporting as safe as well, if you have a minute.

https://global.sitesafety.trendmicro.com/

<img width="1587" alt="image" src="https://user-images.githubusercontent.com/25161793/221403538-6609b115-e545-41ec-9098-192c16b10f2f.png">
